lib/ansible_tower_client/collection.rb

```diff
@@ -6,17 +6,20 @@ def initialize(api, klass = nil)
       @klass = klass
     end
 
-    def all
-      find_all_by_url(klass.endpoint)
+    # @param get_options [Hash] a hash of http GET params to pass to the api request
+    #   e.g. { :order_by => 'timestamp', :name__contains => 'foo' }
+    def all(get_options = nil)
```

Comment:
@durandom - I had some concerns about passing args to the `all` method - but it seems this is the same behavior as ActiveRecord.  So I'm fine with it.